scripts/custom-sort.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports.register = function () {
```

Comment:
for consistent practice, module.exports.register normally place at the end of the file

Reply:
Done